client/tx/factory.go

```diff
@@ -513,7 +513,7 @@ func (f Factory) Prepare(clientCtx client.Context) (Factory, error) {
 	if initNum == 0 || initSeq == 0 {
 		num, seq, err := fc.accountRetriever.GetAccountNumberSequence(clientCtx, from)
 		if err != nil {
-			return fc, err
+			return fc, nil
```

Comment:
I am not a fan of ignoring the error. Could we have a typed error that we check and return an error for any error that isn't that type error?
The code reads weird otherwise.

Reply:
I agree with @julienrbrt here.

Reply:
technically there is no longer an error here. I removed it from the function, since we assume if the account does not exist, the account number will be 0